.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+# See https://pre-commit.com for more information
+repos:
+  - repo: local
+    hooks:
+      - id: black
+        name: black
+        language: system
+        entry: black --check --diff
+        files: \.py[i]?$
```

Comment:
What is the purpose for this line? `black` looks for `.py` files itself. Yes, it should also seek for `.pyi` files, but can it be configured in the `pyproject.toml` file?

Reply:
If you just run `black [DIRECTORY]` it will look for Python files only, yes, but pre-commit will pass it the list of all the files changed, and black will try to work on non-python files too, failing. Like:

```
$ black pyproject.toml 
error: cannot format pyproject.toml: cannot use --safe with this file; failed to parse source file AST: [...]
```

so this argument tells pre-commit to not pass to black any file that is not Python.